js/PulsarUIComponent.js

```diff
@@ -125,32 +141,58 @@ PulsarUIComponent.prototype.initDataTables = function () {
             select = false;
         }
 
-        $this.DataTable({
+        let table = $this.DataTable({
             dom: dom,
             aaSorting: [],
             bAutoWidth: false,
-            buttons: [
-                'selectAll',
-                'selectNone'
-            ],
+            stateSave: true,
+            pageLength: 25,
+            lengthChange: false,
+            buttons: [],
             columnDefs: [
                 { className: 'control', orderable: false, targets: 0 },
-                { "searchable": false, "targets": [0, 1] },
-                { "orderable": false, "targets": [0, 1] }
             ],
             language: {
                 "emptyTable": $this.data('empty-table'),
                 "info": "Showing _START_ to _END_ of _TOTAL_ items",
                 "infoEmpty": 'No items',
                 "infoFiltered": " (filtered from _MAX_ items)",
-                "zeroRecords": "No items matched your filter, please clear it and try again"
+                "zeroRecords": "No items matched your filter, please clear it and try again",
+                "search": "Quick filter"
             },
             select: select,
             stateSave: false
         });
 
         // Add sticky scroll bar
         component.stickyScrollBarComponent.init($this.parent());
+
+        // Init bulk actions menu
+        component.toggleBulkActions($this);
+
+        // Rerun bulk actions on row select
+        table.on('select deselect', function (e, dt, type, indexes) {
+            if (type === 'row') {
+                component.toggleBulkActions($this);
+            }
+        });
+
+            
+        $this.on('click', '.js-select-all', function(e) {
```

Comment:
Really minor (and something we should setup a linter for) we are inconsistent with `function ()` vs `function()`, I prefer the former.

Reply:
Something to add to the linter?

Reply:
We need to add the linter first 😄